src/pybind/ceph_argparse.py

```diff
@@ -492,11 +492,16 @@ class CephFilepath(CephArgtype):
     Openable file
     """
     def valid(self, s, partial=False):
+        cleanup = False
```

Comment:
for `debug dump_missing` the specified file path is a path on the node that the target osd resides on, i think the command should succeed even if the specified file does not exist.
my fix is not an ideal solution either,  since `std::ofstream` will validate it properly, maybe we should not validate the path?
any suggestions?